pkg/apis/monitoring/v1alpha1/types.go

```diff
@@ -168,6 +172,12 @@ type PrometheusConfig struct {
 	EnableRemoteWriteReceiver bool `json:"enableRemoteWriteReceiver,omitempty"`
 }
 
+type AlertmanagerConfig struct {
+	// Disables Alertmanager deployment
+	// +optional
+	DisableAlertmanager bool `json:"disableAlertmanager,omitempty"`
```

Comment:
Do we need to repeat Alertmanager?

```suggestion
	Disabled bool `json:"disabled,omitempty"`
```

Reply:
we don't :)